contracts/test/token/starkgate/behavior/ERC20.ts

```diff
@@ -98,15 +98,19 @@ export const shouldBehaveLikeStarkGateERC20 = (beforeFn: BeforeFn) => {
           amount: uint256.bnToUint256(103),
         })
         throw new Error('This should not pass!')
-      } catch (error: any) {}
+      } catch (error: any) {
+        expect(/assert/gi.test(error.message)).to.be.true
```

Comment:
Can we test for more here than just for the 'assert' string? 
There should be more data/info in that error message, right?